app/assets/stylesheets/application.css.sass

```diff
@@ -9,8 +9,17 @@
   font-size: 75%
   font-weight: bold
 
-// Fixes the background scrolling when opening a modal 
+// Fixes the background scrolling when opening a modal
 body.modal-open
   overflow: hidden
   position: fixed
-  width: 100%
\ No newline at end of file
+  width: 100%
+
+// Add spacing around nav tabs in panel
+.panel
+  >.nav.nav-tabs
+    padding-top: 20px
+    margin-bottom: 10px
+
+  >.panel-body
+    padding: 15px 0
```

Comment:
From what I saw in many places there are cols (ex. `.col-sm-6`) that are not inside of a `.row`. 
Bootstrap relies very much on cols being inside rows. Maybe that was the issue.